packages/hlsjs-playback/src/hls.test.js

```
@@ -1148,6 +1150,96 @@ describe('HlsjsPlayback', () => {
+      const onWarning = vi.fn()
+      playback.on(Events.Custom.PLAYBACK_ERROR_WARNING, onWarning)
+
+      playback._onHLSJSError('hlsError', {
+        fatal: true,
+        type: HLSJS.ErrorTypes.NETWORK_ERROR,
+        details: HLSJS.ErrorDetails.MANIFEST_LOAD_ERROR,
+        response: { code: 500 }
+      })
+
+      expect(onWarning).toHaveBeenCalledTimes(1)
+      expect(onWarning.mock.calls[0][0].level).toBe(PlayerError.Levels.FATAL)
```

ESLint

[error] 'PlayerError' is not defined.

(no-undef)